Updated content for Supported Queries (#2382)
couchbase/docs-server: modules/fts/pages/fts-supported-queries-regexp.adoc

= Regexp Query

A _regexp_ query finds documents containing terms that match the specified regular expression.
Please note that the regex query is a non-analytic query, meaning it won't perform any text analysis on the query text.

[source,json]
----
{
 "regexp": "inter.+",
 "field": "reviews.content"
}
----

> A regexp query is also demonstrated by means of the Java SDK, in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].

Reviewer: ```suggestion
A demonstration of a regexp query using the Java SDK can be found in xref:3.2@java-sdk::full-text-searching-with-sdk.adoc[Searching from the SDK].
```

Author: updated
